src/Catalyst.Core.Modules.Kvm/DeltaExecutor.cs

```diff
@@ -283,11 +241,8 @@ private void Execute(ContractEntry entry, StateUpdate stateUpdate, ITxTracer txT
 
             try
             {
-                if (entry.IsValidDeploymentEntry)
-                {
-                    PrepareContractAccount(env.CodeSource);
-                }
-                
+                if (entry.IsValidDeploymentEntry) PrepareContractAccount(env.CodeSource);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/app/catalyst-network/Catalyst.Framework/pullRequest?prid=4614986)